src/components/pages/Article/article.js

```diff
@@ -1,14 +1,43 @@
 import React from 'react';
+import articleRequests from '../../../helpers/data/articleRequests';
+import authRequests from '../../../helpers/data/authRequests';
 import './article.scss';
 
 class Article extends React.Component {
   render() {
+    const myUid = authRequests.getCurrentUid();
+    const deleteEvent = (e) => {
+      articleRequests.deleteArticle(this.props.id)
+        .then(() => {
+          this.props.updateArticles();
+        })
+        .catch(err => console.error('error with delete single', err));
+    };
+
+    if (this.props.uid === myUid) {
+      return (
+      <div className='card'>
+        <div className='card-body'>
+          <h5 className='card-title'>{this.props.title}</h5>
+          <p className='card-text'>{this.props.synopsis}</p>
+          <a href={this.props.url} className='btn btn-primary'>{this.props.url}</a>
+          <div>
+            <button
```

Comment:
Because the button is the only thing that changes here is whether or not the button is rendered, check out inline conditional rendering: https://blog.logrocket.com/conditional-rendering-in-react-c6b0e5af381e

That way if you had to change the general HTML structure of this, you wouldn't have to do it in 2 places.